### DISCUSSION TITLE
Suggestion: Users / Customer Categorization

### DISCUSSION THREAD

**Discussion by xxScorpius97xx** [swanchain/governance#5] [Ideas]
How if we categorize users/customers within the Swan Chain DAO to better tailor the services and enhance user experience. Example:

**1. Category Based on Engagement:**
   - Contributors: Users who actively participate in governance, development, and decision-making processes.
   - End Users: Individuals or entities primarily using the platform's products and services.
   - Airdrop Hunters: Users mainly engaged during specific events like airdrops, who may have limited long-term engagement.

**2. Category Based on Experience Level:**
   - Newcomers: Users who are new to Swan Chain and blockchain technology, requiring additional guidance and support.
   - Intermediate Users: Those with some experience in Web3 but who may still need occasional assistance.
   - Advanced Users: Experienced individuals who are likely to contribute to technical discussions and governance.

**3. Category Based on Use Cases:**
   - Developers: Individuals or teams building applications on top of Swan Chain.
   - Investors: Users interested in the financial aspects, such as staking and trading tokens.
   - Researchers: Users leveraging the platform for Decentralized Science (DeSci) and other innovative projects.

I believe that categorizing users in these ways could help us better allocate resources, design more effective communication strategies, and ultimately serve community more effectively

**Comment by gugege521** [comment]
这样区分的主要目的是什么呢？

**Comment by flyworker** [comment]
For me, we have 3 classes of contributors:
* Developer
* Computing Provider 
* Participants

Developers are mostly product user and computing resource consumer
Participant are people who involved in daily discussion , it can also be called community member
